Spotify Bluetooth Headset Listener.xcodeproj/project.pbxproj

```diff
@@ -398,7 +398,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Prefix.pch";
 				INFOPLIST_FILE = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Info.plist";
-				MACOSX_DEPLOYMENT_TARGET = 10.6;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
I actually ended up rewrite this to fire media key presses (so it works with any app that supports those) ;)

Please give the new beta version a test:  https://s3-us-west-2.amazonaws.com/jguice/mac-bt-headset-fix-beta/Spotify+Bluetooth+Headset+Listener.zip